spec/index.bs

```diff
@@ -5888,6 +5889,19 @@ dictionary GPUImageCopyTextureTagged : GPUImageCopyTexture {
         2d canvases are [[html#premultiplied-alpha-and-the-2d-rendering-context|always premultiplied]],
         while WebGL canvases can be controlled via <l spec=html>[=WebGLContextAttributes=]</l>.
         {{ImageBitmap}} premultiplication can be controlled via {{ImageBitmapOptions}}.
+
+    : <dfn>originBottomLeft</dfn>
+    ::
+        Describes whether the origin (0, 0) to copy from source image is bottom left corner, or not.
```

Comment:
(The text in this description even says it affects how to read the "source image" so it makes sense to put this option on the source.)